metadata.rb

```diff
@@ -6,4 +6,4 @@
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 version          '0.3.0'
 
-depends 'ark', '~> 0.9.0'
+depends 'ark', '~> 0.8.2'
```

Comment:
sorry, this PR was against the wrong branch. was trying to fix it for our forked cookbook.  There seems to be another PR that fixes this: https://github.com/sit/chef-packer/pull/22 

teh dep cycle was caused by something else in our setup, not packer.